index.html

```diff
@@ -37,5 +37,164 @@ <h4 class="connectmywork"> LET’S CONNECT</h4>
+            </div>
+        <div class="section1"><p > A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+        </div>
+        <div class="skil">
+            <ul class="program2" >
+                <li class="Program-skilhtml">html</li>
+                <li class="Program-skilhtml">css</li>
+                <li class="skiljava">javascript</li>
+            </ul>
+        </div>
+        <div class="buttonsee">
+            <button class="button1"> See Project</button>
+        </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack_03">
+            <div>
+                <a class="card_03" href="#"><img src="./image/Snapshoot Portfolio1 (2).png" alt=""></a>
+            </div>
+            <div class="Multipost">
+                <h3 class="Multi-pos_3"  >Multi-Post Stories</h3>
+            </div>
+            <div class="work_main_3">
+                <ul class="works">
+                    <li class="canopy"> CANOPY</li>
+                    <li class="BackendDev">BackendDev</li>
+                    <li class="num"> 2015 </li>
+                </ul>
+            </div>
+            <div class="section1">
+                <p > A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+            </div>
+            <div class="skil">
+                <ul class="program1">
+                    <li class="Program-skilhtml">html</li>
+                    <li class="Program-skilhtml">css</li>
+                    <li class="skiljava">javascript</li>
+                </ul>
+            </div>
+            <div class="buttonsee">
+                <button class="button1"> See Project</button>
+            </div>
+   
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack_04">
+        <div>
+            <a class="card_04" href="#"><img src="./image/Snapshoot Portfolio1 (3).png" alt=""></a>
+        </div>
+        <div class="tonic_4">
+            <h3  class="tonic" >Tonic</h3>
+        </div>
+        <div class="work_main">
+                <ul class="works">
+                    <li class="canopy"> CANOPY</li>
+                    <li class="BackendDev">BackendDev</li>
+                    <li class="num"> 2015 </li>
+                </ul>
+        </div>
+        <div class="sectiondaily">
+            <p class="section1" > A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+        </div>
+        <div >
+            <div class="skil" >
+                <ul class="program1">
+                    <li class="Program-skilhtml">html</li>
+                    <li class="Program-skilhtml">css</li>
+                    <li class="skiljava">javascript</li>
+                </ul>
+            </div>
+        </div>
+        <div class="buttonsee">
+            <button class="button1"> See Project</button>
+        </div>
+    </div>
+  </section>
+  <section class="Workssection">
```

Comment:
- [ ] **According to the previous code reviewer,** It would be better to display the main container of the projects with a ```display:grid``` but I have realized that you created a ```<section class="Workssection">```  for every card which is not a good practice and you can not use CSS Grid to arrange them. :+1: :+1: 

    Kindly, make sure that all your card are kept with a single ```section``` container that uses ```display:grid``` to display the contents to meet the project requirements :pray: 

_You may check [a Complete Guide to CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) for more info_